New art assets

CHDiaz opened the pull request:
Ignore The char folder, I'll delete it when I'm done with the sprite

CHDiaz committed:
New art for 5/11/15

Transfering to edit on desktop

CHDiaz committed:
Several Additions

-Added Pathfinding over all areas*
-Added to player prefab and Drkntslv objects
*Only tested in town and arena but should work in all in      theory
-Added DGB assets
-Changed Townzone pillar to DGB statue

simonayzman commented:
Don't change the prefab the way you changed it. Create a copy of it and append "Christian" or something. It won't merge otherwise. Also, just make the changes the you'll need to make and I'll accept the pull in an hour.

CHDiaz committed:
New art assets

-Added new chosen model
-Created state machines for animation
-will code animation in the morning
